### PR TITLE
Refactor Welcome page,  Berkeley Upgrade Section and docker-compose examples (by MF)

### PR DESCRIPTION
**Pull request by kaozenn**
Content of this PR:
- Refactor Welcome / Landing Page to accommodate the Hark Fork period (21th May -> 4th Jun)
- Berkeley Upgrade:
  - Moved Archive Migration in its own section and added a docker example
  - Added new articles based on [this document](https://docs.google.com/document/d/1lpxXStu2P1OBeKRhlRfAN54CzDTjZfUu0wFRQn4fgeA/edit#heading=h.2hqz0ixwjk3f) (requirements, upgrade steps, etc...)
- Node Operators
  - Added docker-compose examples for Archive , Seeds, Snark Worker, Block Producer

Note: There are some place holders **_[NEEDS-UPDATE]_** that will need to be updated once the Hard Fork Package are published

Feel free to comment / discuss this PR

### CODE REVIEW

File: docs/berkeley-upgrade/archive-migration/migrating-archive-database-to-berkeley.mdx [hunk 1]
```diff
@@ -272,16 +272,16 @@ where:
 
 ### Stage 2: Incremental migration
 
-After the initial migration, the data is migrated data up to the last canonical block. However, Devnet/Mainnet data is progressing with new blocks that must also be migrated again and again until the fork block is announced. 
+After the initial migration, the data is migrated data up to the last canonical block. However, Devnet/Mainnet data is progressing with new blocks that must also be migrated again and again until the fork block is announced.
 
 :info: Incremental migration can, and probably must, be repeated a couple of times until the fork block is announced by Mina Foundation.
```
Comment (halsaphi) [inline docs/berkeley-upgrade/archive-migration/migrating-archive-database-to-berkeley.mdx:277]: info box not working ... need :::info closed by :::

Comment (kaozenn) [reply docs/berkeley-upgrade/archive-migration/migrating-archive-database-to-berkeley.mdx]: This should be sorted out now.

Comment (halsaphi) [reply docs/berkeley-upgrade/archive-migration/migrating-archive-database-to-berkeley.mdx]: Also see **info** on line 424 - should this also be a "box" by using :::info ::: ?

**Review by dkijania — APPROVED** [review]

File: docs/berkeley-upgrade/appendix.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+} else {
+  let text = await response.text();
+  console.log('Error:\n', text);
+}
+
+function objectToGraphqlQuery(obj: any) {
+  let json = JSON.stringify(obj, null, 2);
+  // removes the quotes on JSON keys
+  return json.replace(/\"(\S+)\"\s*:/gm, '$1:');
+}
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:80]:
```suggestion
const sendPaymentMutationQuery = `
mutation SendPayment($input: SendPaymentInput!, $signature: SignatureInput!) {
  sendPayment(input: $input, signature: $signature) {
    payment {
      hash
    }
  }
}
`;
const graphQlVariables = {
  input: signedPayment.data,
  signature: signedPayment.signature,
};
const body = JSON.stringify({
  query: sendPaymentMutationQuery,
  variables: graphQlVariables,
  operationName: 'SendPayment',
});

const paymentResponse = await fetch(url, {
  method: 'POST',
  headers: { 'Content-Type': 'application/json' },
  body
});

const paymentResponseJson = await paymentResponse.json();
if (paymentResponse.ok) {
  console.log(`Transaction hash: ${paymentResponseJson.data.sendPayment.payment.hash}`);
} else {
  console.error(JSON.stringify(paymentResponseJson));
}

```

File: docs/berkeley-upgrade/appendix.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## Migration from o1labs/client-sdk to mina-signer
+
+The signing library o1labs/client-sdk was deprecated some time ago and will stop working after the upgrade. All users should upgrade with the library mina-signer https://www.npmjs.com/package/mina-signer.
+
+Below you will find an example of how to use the mina-signer library. Please keep in mind the following:
+
+1. Make sure to adjust the nonce to the correct nonce on the account you want to use
+2. Use fee 1 MINA to get the TX submitted quickly
+3. Update the “url” variable with an existing Mina Node GraphQL
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:22]:
```suggestion
3. Update the `url` variable with an existing Mina Node GraphQL endpoint
```

File: docs/berkeley-upgrade/appendix.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  - Berkeley
+  - upgrade
+  - appendix
+---
+
+# Appendix
+
+## Migration from o1labs/client-sdk to mina-signer
+
+The signing library o1labs/client-sdk was deprecated some time ago and will stop working after the upgrade. All users should upgrade with the library mina-signer https://www.npmjs.com/package/mina-signer.
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:16]:
```suggestion
The signing library `o1labs/client-sdk` was deprecated some time ago and will stop working after the Mina mainnet upgrade. All users should upgrade to use the [mina-signer](https://www.npmjs.com/package/mina-signer) library.
```

File: docs/berkeley-upgrade/appendix.mdx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Below you will find an example of how to use the mina-signer library. Please keep in mind the following:
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:18]:
```suggestion
Below you will find an example of how to use the `mina-signer` library. Please keep in mind the following:
```

File: docs/berkeley-upgrade/appendix.mdx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Make sure to adjust the nonce to the correct nonce on the account you want to use
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:20]:
```suggestion
1. Make sure to adjust the `nonce` to the correct nonce on the account you want to use as "sender"
```

File: docs/berkeley-upgrade/appendix.mdx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+2. Use fee 1 MINA to get the TX submitted quickly
+3. Update the “url” variable with an existing Mina Node GraphQL
+
+```javascript
+import { Client } from 'mina-signer';
+
+// create testnet client and define hard-coded keypair
+
+const client = new Client({ network: 'testnet' });
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:29]:
```suggestion
const client = new Client({ network: 'testnet' }); // Mind the `network` client configuration option
```

**Review by shimkiv — CHANGES_REQUESTED** [review]

File: docs/berkeley-upgrade/appendix.mdx [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+# Appendix
+
+## Migration from o1labs/client-sdk to mina-signer
+
+The signing library o1labs/client-sdk was deprecated some time ago and will stop working after the upgrade. All users should upgrade with the library mina-signer https://www.npmjs.com/package/mina-signer.
+
+Below you will find an example of how to use the mina-signer library. Please keep in mind the following:
+
+1. Make sure to adjust the nonce to the correct nonce on the account you want to use
+2. Use fee 1 MINA to get the TX submitted quickly
```
[the remark this answers is not in the document]
Comment (kaozenn) [reply docs/berkeley-upgrade/appendix.mdx]: I have removed the line

**Review by kaozenn — COMMENTED** [review]
@shimkiv I have included all your suggestions

File: docs/berkeley-upgrade/appendix.mdx [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+# Appendix
+
+## Migration from o1labs/client-sdk to mina-signer
+
+The signing library `o1labs/client-sdk` was deprecated some time ago and will stop working after the Mina mainnet upgrade. All users should upgrade to use the [mina-signer](https://www.npmjs.com/package/mina-signer) library.
+
+Below you will find an example of how to use the `mina-signer` library. Please keep in mind the following:
+
+1. Make sure to adjust the `nonce` to the correct nonce on the account you want to use as "sender"
+2. Update the “url” variable with an existing Mina Node GraphQL
```
Comment (shimkiv) [inline docs/berkeley-upgrade/appendix.mdx:21]:
```suggestion
1. Update the `url` variable with an existing Mina Node GraphQL endpoint
```

**Review by shimkiv — APPROVED** [review]

**Pull request merged**